### DISCUSSION TITLE
Empty space bug

### DISCUSSION THREAD

**Discussion by schmelto** [community/community#41237] [Mobile] [answered]
When opening issues or pull request on mobile I got following empty space 

https://user-images.githubusercontent.com/30869493/206811342-9bf230bf-7378-420c-9be5-eae55fa641e4.MOV

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.92.0 (85285908)
Device: iPhone 13 mini (iOS 16.1.2)
</details>

**Comment by eliperkins** [comment, marked as answer]
Hey @schmelto , thanks for this bug report! Having all of that extra spacing sure is frustrating, especially when you're trying to get the content.

I've shared the details with our engineering team and I think we should have a bug fix for this coming out soon. Stay tuned!

**Reply by schmelto** [reply]
Thanks @eliperkins 
I will stay tuned!

**Reply by schmelto** [reply]
Just saw in the new released GitHub mobile version this bug was fixed

**Reply by eliperkins** [reply]
Great eye! A bug fix for this is rolling out on the App Store now in 1.95.0 (88456997). Please let me know if you're continuing to run into this bug after updating. I'm happy to dive back in if this continues to be an issue.
